src/lib/paginator/paginator-intl.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Injectable} from '@angular/core';
+
+/**
+ * Paginator labels that require internationalization. To modify the labels and text displayed,
+ * extend this class with custom values and inject it as a custom provider.
+ */
+@Injectable()
+export class MdPaginatorIntl {
```

Comment:
Perhaps we should have an interface that the user can implement when overriding?